src/stories/lib/markdown-to-csf.js

```diff
@@ -29,39 +29,20 @@ const processor = unified()
 
 export function markdownToCsfWithDocsPage (markdownText) {
 
-  const markdownAst = processor.parse(markdownText);
-
-  const frontmatterNode = markdownAst.children.find((node) => node.type === 'yaml');
-  const kind = getKind(frontmatterNode);
-
-  const headingNode = markdownAst.children.find((node) => node.type === 'heading' && node.depth === 1);
-  const subtitle = getSubTitle(frontmatterNode, headingNode);
-
-  const title = [kind, subtitle].filter((a) => a != null).join('/');
-
   const htmlContent = processor().processSync(markdownText).contents;
   const parsedHTML = JSON.stringify(htmlContent);
 
   const csfScript = `
     import { MarkdownDocs } from '/src/stories/lib/markdown-docs.jsx';
     import React from 'react';
```

Comment:
**note**: even though we have removed `react` from our deps, vite still resolves it from `storybook` packages.

I don't see why it would stop working unless Storybook stops using react (but in such case, we wouldn't even need this file anyway :smile:).